entries/entry04.md

```diff
@@ -1,7 +1,31 @@
-# Entry 4
-##### X/X/XX
+# Entry 4: Swift
+##### 2/9/20
+
+### What was done after learning cocoapods?
+
+I began to learn a bit more about User Interfaces or UI.
+It seemed like something fun and important to learn.
+
+##### Why is it important to learn?
+It is important to learn about what will be simple to use. We can't make an app that will then be difficult/complex to use.
+
+I used [the official apple website](https://developer.apple.com/library/archive/referencelibrary/GettingStarted/DevelopiOSAppsSwift/BuildABasicUI.html) to start my little research (as I will not focus on this matter for too long).
+
+I ended up coming up with a little list to help me when making the UI:
+
+1. Can the user use the app with little difficulty?
+2. Does the interface allow the user to complete their task in a timely manner?
+3. After leaving the app, how likely will the user be able to remember how to use it the next time they open and use the app?
+
+It is not much but it definitely does help in making sure that we do not go overboard with how we place the calendar, the buttons and functions of the app.
+
+I definitely did learn more about how other people will be able to interact with the app. From having to make sure that the interface is simple and efficient, to making sure that they understand what it is that they are doing with the app.
+
+The little specs like color, design, small animations. That I can deal with later. I only focused on UI right now, so when we get to it (or rather when we start making the functions of the buttons) we are not completely lost.
+
+I did try making something on my Mac computer, but because of technical issues I was not able to finish or get screenshots of the code and the temporary outcome. Once those issues are fixed, this entry will be updated.
```

Comment:
Make sure you get these with enough time ahead of time! This is what really shows evidence of learning (knowledge).